lib/RoadizUserBundle/README.md

```diff
@@ -211,8 +211,23 @@ services:
 
 ### Override login link URL
 
-Decorate `Symfony\Component\Security\Http\LoginLink\LoginLinkHandlerInterface` service if you need to generate a login-link
-with a different **base-uri**, for example if you are using a different domain for your frontend application.
+A service was enable for generate a login-link  with a different **base-uri**,
+for example if you are using a different domain for your frontend application.
+
+```yaml
+# config/services.yaml
+services:
+    App\Security\LoginLinkHandler:
```

Comment:
Be careful, we instruct projects developer to use our FrontendLoginLinkHandler, not their anymore

```suggestion
    RZ\Roadiz\UserBundle\Security\Security\FrontendLoginLinkHandler:
```